process/block/metablock.go

```diff
@@ -42,6 +43,18 @@ type metaProcessor struct {
 	nextKValidity uint32
 
 	chRcvAllHdrs chan bool
+
+	appStatusHandler core.AppStatusHandler
```

Comment:
since both shard processor and meta processor have an appStatusHandler pointer, might move this in base processor

Reply:
Done. Removed appStatusHandler from metablock and shardblock and added it in baseProcessor.